docs/hdfs_connector.rst

```diff
@@ -95,13 +150,17 @@ You should see the following output::
 
 .. note:: If you want to run the Quickstart with Hive integration, before starting the connector,
```

Comment:
Not being as familiar with the quickstart, I thought the transition to talk about Hive was a little weird as it stands. Understand that it's not related to the CLI, but it does seem like the transition could be made smoother. It's almost like "If you're using Hive, start over and change the connector config slightly, then see the data in Hive a bit differently." 

Alternatively, perhaps the "If you're using Hive" note could be moved up earlier around when the connector configuration is first discussed. Not sure if that's better or worse.